lib/daru/dataframe.rb

```diff
@@ -341,11 +345,6 @@ def initialize source, opts={}
       update
     end
 
-    def vector(*)
-      $stderr.puts '#vector has been deprecated in favour of #[]. Please use that.'
-      self[*names]
-    end
-
```

Comment:
I'm removing it completely because there was long-sitting bug in it (from where the `names` coming, ha?) and nobody objected. Therefore, nobody uses it!